meetings/2023-12-05.md

```diff
@@ -0,0 +1,219 @@
+# W3C Solid Community Group and Social Community Group Joint Meeting
+
+* Date: 2023-12-05T16:00:00Z
+* Call: https://meet.jit.si/solid-cg-social-cg
+* Status: Published
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* [Virginia Balseiro](https://virginiabalseiro.com/#me)
+* [Ted Thibodeau Jr](https://github.com/TallTed) (he/him) (OpenLink Software)
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* Maxime Lecoq
+* [Michiel de Jong](https://michielbdejong.com)
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+* [Sébastien Rosset](https://activitypods.org)
+* [Jeff Zucker](mailto:dubzed@gmail.com)
+* [Aaron Coburn](https://id.inrupt.com/acoburn)
+* [Tantek Çelik](https://tantek.com/)
+* [Matthias Evering](https://solidweb.me/testpro/)
+* [Emelia Smith](https://hachyderm.io/@thisismissem)
+* [Tim Berners-Lee](https://timbl.inrupt.net/profile/card#me)
+* [Pierre-Antoine Champin](https://solid.champin.net/pa/profile/card#me)
+* Juan Caballero (bumblefudge)
+* Philippe Le Hegaret
+* [Angelo Gladding](https://ragt.ag)
+* Dmitri Zagidulin
+* Bob Wyman
+* a
```

Comment:
```suggestion
```

Reply:
It is not a typo :) Someone with that name was in the call, and I've seen them before in other Social CG calls.